var/spack/repos/builtin/packages/libjpeg-turbo/package.py

```diff
@@ -34,7 +34,9 @@ class LibjpegTurbo(AutotoolsPackage):
     homepage = "http://libjpeg-turbo.virtualgl.org"
     url      = "http://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-1.3.1.tar.gz"
 
-    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257')
+    version('1.5.1', '55deb139b0cac3c8200b75d485fc13f3')
+    version('1.5.0', '3fc5d9b6a8bce96161659ae7a9939257', preferred=True)
```

Comment:
As I did not know which other sw were depending on libjpeg-turbo, I would not like to trigger recompilation on and eventually incompatibilities on other packages.
I do not know what is the best practice: 
Personally I' m a bit annoyed when  a new version is added to a low level library and this trigger ( bad ) recompilation in other upper level packages
In order to avoid this, I' ve set up the preferred as the old one... 
For example 
[This](https://github.com/LLNL/spack/pull/3043/commits/64c2817842a902275635e80540ae4845b72f2bcf) has been recently triggered by new openssl 1.1